src/styles/index.scss

```diff
@@ -1,3 +1,254 @@
+
 body {
   margin: 0;
 }
+
+a {
+  color: inherit;
+  text-decoration: none;
+}
+
+// #region header
+
+.header {
+  display: flex;
+  padding: 0 50px;
+  justify-content: space-between;
+
+  .logo {
+    height: 40px;
+    width: 40px;
+    object-fit: contain;
+    padding: 10px 0;
+  }
+
+  nav {
+    text-transform: uppercase;
+
+    .nav__list {
+      display: flex;
+      margin: 0;
+      padding: 0;
+      list-style: none;
+
+      .nav_item {
+        transition: all ease-in-out 0.15s;
+        color: $nav-link-color;
+
+        &:not(:last-child) {
+          margin-right: 20px;
+        }
+
+        &:hover {
+          color: $hover-color;
+        }
+
+        .nav_link {
+          font-weight: 500;
+          font-size: 12px;
+          display: inline-block;
+          height: $header-height;
+          line-height: $header-height;
+        }
+
+        .is-active {
+          position: relative;
+          color: $hover-color;
+
+          &::after {
+            content: '';
+            position: absolute;
+            height: 4px;
+            background: $hover-color;
+            border-radius: 8px;
+            width: 100%;
+            bottom: 0;
+            left: 0;
+          }
+        }
+      }
+    }
+  }
+}
+
+// #endregion header
+
+
+// #region main
+
+// #region cards container (media)
+.cards__container {
+  padding: 50px 40px;
+  display: grid;
+
+  grid-template-rows: 1fr 1fr;
+  grid-template-columns: repeat(var(--column-count), 200px);
+  gap: 46px 48px;
+
+  justify-items: center;
+  justify-content: center;
+}
+
+@media (min-width: 488px) {
+  .cards__container{
+    --column-count: 2;
+  }
+}
+
+@media (min-width: 768px) {
+  .cards__container{
+    --column-count: 3;
+  }
+}
+
+@media (min-width: 1024px) {
+  .cards__container{
+    --column-count: 4;
+  }
+}
+// #endregion cards container
+
+// #region card
+
+.product__card {
+  box-sizing: border-box;
+  border-radius: 5px;
+  border: 1px solid rgb(243, 243, 243);
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  transition: all 0.3s ease-in-out;
+
+  &:hover {
+    transform: scale(120%);
+  }
+
+  &:hover .product__name {
+    color: #34568b;
+  }
+
+  .product__image {
+    margin: 32px 20px 40px 20px;
+    height: 134px;
+    background: url(../images/imac.jpeg) center no-repeat;
+    background-size: contain;
+  }
+
+  .product__info {
+    display: flex;
+    flex-direction: column;
+    padding: 0 16px 16px 16px;
+
+    .product__name {
+      padding-bottom: 4px;
+      font-size: 12px;
+      font-weight: 500;
+      line-height: 18px;
+    }
+
+    .product__code {
+      padding-bottom: 16px;
+      color: $product-att-color;
+      font-size: 10px;
+      font-weight: 400;
+      line-height: 14px;
+    }
+
+    .product__review-box {
+      width: 100%;
+      display: flex;
+      justify-content: space-between;
+      padding-bottom: 24px;
+
+      .product__stars {
+        display: flex;
+
+        .product__star {
```

Comment:
Avoid hardcoding image paths in the SCSS file. Consider passing image paths as variables or using a function/mixin for setting background images.